packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -399,6 +399,16 @@ class FileCollection extends DocumentCollection {
     forceFileDownload(`${href}?Dl=1`, filenameToUse)
   }
 
+  /**
+   * Get the binary of a file or a specific version of a file
+   * used for Mobile native download
```

Comment:
I am not sure we should write in the doc for which usage a function is here. It is probable that at some point it is used in another place (or that the mobile native download does not use it anymore) and the comment will no tell the truth anymore.

Reply:
I like having an exemple of why I should use that method instead of, let's see for instance... `download()`. 

We have 3 methods to download a file from the stack, and I'm just figuring why (after few months). 

` Get the binary of a file or a specific version of a file.
Useful for instance when you can't download the file directly (via a content-disposition attachement header) and need to store it before doing an operation.`

Is it better?